src/reports/biospecimen-request/generateBiospecimenRequestFiles.ts

```diff
@@ -43,50 +43,55 @@ export default async function generateFiles(
 
     const searchParams = await makeReportQuery(extendedConfig, sqon, wantedFields, userId, accessToken);
 
-    console.time(`biospecimen request search`);
-    await executeSearchAfterQuery(es, normalizedConfigs.alias, searchParams, {
-        onPageFetched: rawChunk => {
-            rawChunk.forEach(row => {
-                const study_code = (row as any).study.study_code;
-                if (!workSheets.has(study_code)) {
-                    // Add study sheet in wb
-                    const newSheetConfig = generateStudyTab(study_code);
-                    const newSheet = wb.addWorksheet(newSheetConfig.sheetName);
-                    newSheetConfig.columns.forEach(addHeaderCellByType(newSheet));
-                    workSheets.set(study_code, newSheet);
-                    workSheetConfigs.set(study_code, newSheetConfig);
-                    workSheetWrappers.set(study_code, { rowIndex: 2 });
+    try {
+        console.time(`biospecimen request search`);
```

Comment:
I would put that console.time just before the try and console.timeEnd in a finally clause.